src/Parser/Warrior/Arms/Modules/BuffDebuff/ColossusSmashUptime.js

```diff
@@ -34,6 +34,7 @@ class ColossusSmashUptime extends Module {
         icon={<SpellIcon id={SPELLS.COLOSSUS_SMASH_DEBUFF.id} />}
         value={`${formatPercentage(colossusSmashUptime)} %`}
         label="Colossus Smash uptime"
+        tooltip={`Colossus Smash increases your dealt damage by 15% + Mastery against that debuffed target.`}
```

Comment:
Seems unnecessary to me as it's basically explaining what the spell does which is already in the spell's tooltip (excluding the mastery bit, but I guess all Arms Warriors (that use this analyzer) know what their spells and mastery do).